From a371cfd6c9d77f3bfed5016148c4e86ea6fedfa8 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Wed, 30 Nov 2011 16:23:19 +0800
Subject: ENGR00163615 mx6q: sabrelite RevC: change GPIO for flexcan

GPIOs used by flexcan are changed on RevC board.
Updating the code for RevC board.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabrelite.c |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabrelite.c b/arch/arm/mach-mx6/board-mx6q_sabrelite.c
index 589a57b..118b6e0 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabrelite.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabrelite.c
@@ -80,8 +80,8 @@
 #define MX6Q_SABRELITE_USB_OTG_PWR	IMX_GPIO_NR(3, 22)
 #define MX6Q_SABRELITE_CAP_TCH_INT1	IMX_GPIO_NR(1, 9)
 #define MX6Q_SABRELITE_USB_HUB_RESET	IMX_GPIO_NR(7, 12)
-#define MX6Q_SABRELITE_CAN1_STBY	IMX_GPIO_NR(1, 25)
-#define MX6Q_SABRELITE_CAN1_EN		IMX_GPIO_NR(1, 27)
+#define MX6Q_SABRELITE_CAN1_STBY	IMX_GPIO_NR(1, 2)
+#define MX6Q_SABRELITE_CAN1_EN		IMX_GPIO_NR(1, 4)
 
 #define MX6Q_SABRELITE_SD3_WP_PADCFG	(PAD_CTL_PKE | PAD_CTL_PUE |	\
 		PAD_CTL_PUS_22K_UP | PAD_CTL_SPEED_MED |	\
@@ -106,9 +106,9 @@ static iomux_v3_cfg_t mx6q_sabrelite_pads[] = {
 	/* CAN1  */
 	MX6Q_PAD_KEY_ROW2__CAN1_RXCAN,
 	MX6Q_PAD_KEY_COL2__CAN1_TXCAN,
-	MX6Q_PAD_ENET_CRS_DV__GPIO_1_25,	/* STNDBY */
-	MX6Q_PAD_ENET_RXD1__GPIO_1_26,		/* NERR */
-	MX6Q_PAD_ENET_RXD0__GPIO_1_27,		/* Enable */
+	MX6Q_PAD_GPIO_2__GPIO_1_2,		/* STNDBY */
+	MX6Q_PAD_GPIO_7__GPIO_1_7,		/* NERR */
+	MX6Q_PAD_GPIO_4__GPIO_1_4,		/* Enable */
 
 	/* CCM  */
 	MX6Q_PAD_GPIO_0__CCM_CLKO,		/* SGTL500 sys_mclk */
-- 
1.7.9.5

